handlers/lockrenewer.go

```diff
@@ -0,0 +1,64 @@
+package handlers
+
+import (
+	"context"
+	"time"
+
+	servicebus "github.com/Azure/azure-service-bus-go"
+	"github.com/Azure/go-shuttle/tracing"
+	"github.com/devigned/tab"
+)
+
+var _ servicebus.Handler = (*PeekLockRenewer)(nil)
+
+// LockRenewer abstracts the servicebus subscription client where this functionality lives
+type LockRenewer interface {
+	RenewLocks(ctx context.Context, messages ...*servicebus.Message) error
+}
+
+// PeekLockRenewer starts a background goroutine that renews the message lock at the given interval until Stop() is called
+// or until the passed in context is canceled.
+// it is a pass through handler if the renewalInterval is nil
+type PeekLockRenewer struct {
```

Comment:
Did you never complete the peeklock package PR or just wiping it in another PR. I assum not much has changed here.

Reply:
Looks like it's there. Git rm it or some reason to wait?